fastlane/metadata/android/et-EE/short_description.txt

```diff
@@ -0,0 +1 @@
+Kergekaal meediaesitaja tehtud libmpv-ist.
```

Comment:
I think this isn't supposed to contain a period, so not a full sentence
(unless that's an abbreviation)

Reply:
Hello! Thank you for reviewing this file.

I will fix the last word which means "-ist" will be removed by necessary and make simple.

- Martin Eesmaa

Reply:
What about the period `.` at the end?

Reply:
Oh yeah, sorry I forgot to remove dot point at the end. Thanks for reminding me!

- Martin Eesmaa

Reply:
Fixed by removing dot point at the end.